.gitignore

```diff
@@ -1,2 +1,6 @@
 /dist/
 *.out
+.cabal-sandbox/
+.idea/
```

Comment:
The first one is for cabal sandbox temporary files. 
The second is for Intellij IDE.
I added this so this file won't be committed to the repository.
I hope it is ok since those files should never be added to the git repo?

BTW I though about using stack instead of cabal sandbox, but that's probably should be done in separate task.